package.json

```diff
@@ -14,6 +14,11 @@
     "module2_task1": "nodemon --exec babel-node ./module_2/task1",
     "module2_task2": "nodemon --exec babel-node ./module_2/task2",
     "module3": "nodemon --experimental-json-modules ./module_3/main.js",
+    "start": "nodemon --experimental-json-modules ./module_4/main.js",
+    "migration up all": "sequelize-cli db:migrate",
+    "migration revert all": "sequelize-cli db:migrate:undo:all",
+    "seeds up all": "sequelize-cli db:seed:all",
```

Comment:
`seeds:up-all`  or `seeds:up:all` and the same blelow and above - replace, please,  space with `:`